agent/runner/jobs/pbm_helpers.go

```diff
@@ -325,6 +324,8 @@ func waitForPBMRestore(ctx context.Context, l logrus.FieldLogger, dbURL *url.URL
 		name = restoreInfo.Name
 	}
 
+	l.Infof("waiting for pbm restore: %s", restoreInfo.Name)
```

Comment:
```suggestion
	l.Infof("waiting for pbm restore: %s", name)
```

`restoreInfo.Name` is not always valid.